node/src/Transaction.ts

```diff
@@ -115,8 +115,16 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
      * @internal
      */
     readonly commands: redis_request.Command[] = [];
+    readonly set_commands_indexes: number[] = [];
```

Comment:
It should be clear that it is for internal usage only. Please add a documentation and mark it with @internal too.

Reply:
still missing documentation

Reply:
weird, I remember adding it

Reply:
note: nodejs naming convention is lowerCamelCase
setCommandsIndexes